pyvista/examples/downloads.py

```diff
@@ -7631,3 +7631,46 @@ def download_whole_body_ct_female(load=True):  # pragma: no cover
     target_file='s1380',
     load_func=_whole_body_ct_load_func,
 )
+
+
+def download_room_cff(load=True):  # pragma: no cover
+    """Download a room model in CFF format.
+
+    Parameters
+    ----------
+    load : bool, default: True
+        Load the dataset after downloading it when ``True``.  Set this
+        to ``False`` and only the filename will be returned.
+
+    Returns
+    -------
+    pyvista.MultiBlock or tuple
+        DataSet or tuple of filenames depending on ``load``.
+
+    Examples
+    --------
+    >>> import pyvista as pv
+    >>> from pyvista import examples
+    >>> blocks = examples.download_room_cff()
+    >>> mesh = blocks[0]
+    >>> mesh.plot(cpos="xy", scalars="SV_T")
+
+    .. seealso::
+
+        :ref:`Room Cff Dataset <room_cff_dataset>`
+            See this dataset in the Dataset Gallery for more info.
+
+    """
+    return _download_dataset(_dataset_room_cff, load=load)
+
+
+def _dataset_room_cff_files_func():
+    cas = _SingleFileDownloadableDatasetLoader('FLUENTCFF/room.cas.h5')
+    dat = _DownloadableFile('FLUENTCFF/room.dat.h5')
+    return cas, dat
+
+
+_dataset_room_cff = _MultiFileDownloadableDatasetLoader(
+    _dataset_room_cff_files_func,
+    load_func=_load_as_multiblock,
```

Comment:
```suggestion
    _dataset_room_cff_files_func
```

This dataset is already a multiblock, i.e. the new CFF reader (obtained by the loader using pv.read by default) will read this dataset as a multiblock. 
Using `load_as_multiblock` will load all datasets _into_ a multiblock. So, this will nest an existing multiblock into another.

Reply:
Thanks, it seems I need to understand more about the framework.

Reply:
> Thanks, it seems I need to understand more about the framework :)

The loaders can be confusing and there is lots of room for improvement with the API. E.g. sometimes you specify `read_func` but not `load_func`, sometimes it's the other way around, or sometimes you need both and sometimes you don't need either!  In other cases, you can even use a `SingleFile...` loader to load multiple files 😅